QuizAppNew/Model/QuestionModel.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  QuestionModel.swift
+//  QuizAppNew
+//
+//  Created by Jannatun Nahar Papia on 20/8/22.
+//
+
+import SwiftUI
+
+//MARK: Codeble Model....
+
```

Comment:
Why this model? Why need it? please remove this model. you are responsible for making UI design.

Reply:
okay I understand